coldfront/core/user/views.py

```diff
@@ -97,29 +97,31 @@ class UserProfileUpdate(TemplateView):
 
     def post(self, request, *args, **kwargs):
```

Comment:
You might also consider inheriting from `FormView` instead of `TemplateView` so that field-specific errors appear right under the field. There are plenty of examples of this in the repo.

Reply:
noice!